### ISSUE TITLE
Should we show the examples?

### ISSUE THREAD

**Issue by pbugnion** [facultyai/dash-bootstrap-components#130]
Web app examples are particularly effective at demonstrating what a library can do, and how to do it. Since we have some great examples, it would be beneficial to host them somewhere.

We don't really want to host one different app for each example, though. One idea might be to use different base paths for different apps? E.g.

```py
server = Flask(__name__)

app_iris = Dash(__name__, server, routes_pathname="/iris/" # etc.)
app_faithful = Dash(__name__, server, routes_path_name="/faithful/", # etc.)
```

I'm not sure whether we can get the above to work.

An alternative would be to have each app running on a different port with either a single webserver in front that loads apps in iframes or a load balancer that proxies to a different upstream depending on the path. I don't know how easy either of these alternatives would be to deploy.

**Comment by pbugnion** [comment]
Happy to put more thought into this if we think exposing the examples is useful.

**Comment by pbugnion** [comment]
Some useful links here: https://community.plot.ly/t/hosting-multiple-dash-apps-uwsgi-emperor-vs-multi-page-apps-vs-multi-app-projects/6561

**Comment by tcbegley** [comment]
I think this is a great idea.

I don't have any great ideas on deployment right now, but wanted to add a [link to the shiny gallery](https://shiny.rstudio.com/gallery/) which I think does a nice job of showing off example apps.

**Comment by jmsblah** [comment]
I am trying to run the example app in Navbars.py, but I am getting the error: `ImportError: cannot import name '_components' from 'dash_bootstrap_components'  (\dash-bootstrap-components\dash_bootstrap_components\__init__.py)`

**Comment by tcbegley** [comment]
Resolved by #332
